package.json

```diff
@@ -19,20 +19,21 @@
   },
   "build": {
     "appId": "org.ssbc.patchbay",
-    "asarUnpack": [
-      "node_modules/sodium-native/prebuilds"
-    ],
-    "npmSkipBuildFromSource": true,
```

Comment:
why did you remove these two options @cryptix ?

is it because asarUnpack by default hits the node_modules?
I would have thought skipBuiild... would be good because it gets you prebuilts?

Reply:
I've added these back in for the moment